UnitTests/PInvoke/Shared/WinError/HRESULTTests.cs

```diff
@@ -50,6 +50,10 @@ public void GetExceptionTest()
 		Assert.That(new HRESULT(HRESULT.CO_E_ATTEMPT_TO_CREATE_OUTSIDE_CLIENT_CONTEXT).GetException(), Is.TypeOf<COMException>());
 		Assert.That(new HRESULT(HRESULT.E_INVALIDARG).GetException(), Is.TypeOf<ArgumentException>());
 		Assert.That(new HRESULT(HRESULT.E_INVALIDARG).GetException("Bad"), Has.Message.EqualTo("Bad"));
+		#if NETCOREAPP3_0_OR_GREATER
+		var win32HResult = new Win32Error(Win32Error.ERROR_SHARING_VIOLATION).ToHRESULT();
```

Comment:
this is the particular hresult I found the issue with. It returns a [`FileLoadException`](https://learn.microsoft.com/en-us/dotnet/api/system.io.fileloadexception) which has default hresult `COR_E_FILELOAD 0x80131621`, which differs from this `win32HResult` value `0x80070020`

Reply:
Both of these HResult values can be seen here: https://source.dot.net/#System.Threading/src/libraries/Common/src/System/HResults.cs